spec/database-spec/simpleQueries.spec.js

```diff
@@ -0,0 +1,97 @@
+const db = require('../../database/database.js');
+
+beforeAll(() =>
+  // Clears the database
+  // Jest will wait for this promise to resolve before running tests.
+  db.complex.dropAllTables()
+    .then(db.complex.initializeDB)
+);
+
+
+// tests queries for rooms
+describe('Room table populates and returns data', () => {
+  // add some test data before each test
+  beforeEach(() =>
+    db.rooms.drop()
+      .then(db.rooms.create)
+      .then(() => (
+        db.rooms.add({ roomName: 'Flosten Paradise' })
+    ))
+  );
+
+  it('should have one entry populated', () => (
+    db.rooms.all().then((results) => {
+      expect(results.length).toEqual(1);
+    })
+  ));
+
+  it('return an id of 2 when a second room is added', () => (
+    db.rooms.add({ roomName: 'RoomName' }).then((results) => {
+      expect(results).toEqual(2);
+    })
+  ));
+
+  it('should find the id for a given roomName', () => (
+    db.rooms.findId({ roomName: 'Flosten Paradise' }).then((results) => {
+      expect(results.id).toEqual(1);
+    })
+  ));
+
+  it('should find the name for a given roomId', () => (
+    db.rooms.findName({ roomId: 1 }).then((results) => {
+      expect(results.name).toEqual('Flosten Paradise');
+    })
+  ));
+});
+
+
+// Tests queries for users
+describe('Users table populates and returns data', () => {
+  // add some test data before each test
+  beforeEach(() =>
+    db.users.drop()
+      .then(db.users.create)
+      .then(() => (
+        db.users.add({
+          name: 'Corbin',
+          email: 'Dallas',
+          fbToken: 58008
+        })
+    ))
+  );
+
+  it('should have one entry populated', () => (
+    db.users.all().then((results) => {
+      expect(results.length).toEqual(1);
+    })
+  ));
+
+  it('return an id of 2 when a second user is added', () => (
+    db.users.add({
+      name: 'Multipass',
+      email: 'Multi-pass',
```

Comment:
No reason, except that I love The Fifth Element and putting an '@' and a '.com' would have ruined the multipass flow.